build/transpile.js

```diff
@@ -250,8 +250,10 @@ class Transpiler {
             [ /\.isPostOnly\s/g, '.is_post_only'],
             [ /\.reduceFeesByCurrency\s/g, '.reduce_fees_by_currency'],
             [ /\.omitZero\s/g, '.omit_zero'],
-            [ /\.currencyStructure\s/g, '.currency_structure'],
             [ /\.safeCurrencyStructure\s/g, '.safe_currency_structure'],
+            [ /\.precisionModeIsTickSize\s/g, '.precision_mode_is_tick_size'],
+            [ /\.precisionModeIsDecimalPlaces\s/g, '.precision_mode_is_decimal_places'],
+            [ /\.precisionModeIsSignificantDigits\s/g, '.precision_mode_is_significant_digits'],
```

Comment:
@ttodua don't forget to update here as well 😅

Reply:
yeah updated too at this moment :)

Reply:
I've used short names, because per your above suggestion there were `isTickSizePrecision` and `isDecimalPrecision` so, we should either use both in full names:
- isTickSizePrec...
- isDecimalPlacesPrec...

or with first word:
- isTickPrec..
- isDecimalPrec...